### PR TITLE
Pop up alert improvements

### PR DESCRIPTION
**Pull request by Pinkdesu**
- Refactor alert's styles

### CODE REVIEW

**Review by osipov-mit — APPROVED** [review]

File: website/frontend/src/assets/images/alert/error.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (codev0) [inline website/frontend/src/assets/images/alert/error.svg:1]: could you try reduce icon sizes trough [SVGOMG](https://jakearchibald.github.io/svgomg/)?

File: website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
+@use 'assets/scss/variables' as *;
+
+.alert {
+  position: relative;
+  width: 100%;
+  padding: 15px 40px 15px 15px;
+  background-color: $mainBackground;
+  border-radius: 2px;
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss:8]:
```suggestion
  border-radius: $borderRadius;
```

**Review by EugenWay — APPROVED** [review]

File: website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.closeBtn {
+   position: absolute;
+   top: 10px;
+   right: 10px;
+
+   & > img {
+      width: 11px;
+      height: 11px;
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss:23]:
```suggestion
      @use 'assets/scss/mixins' as *;
      @include square(11px);
```

File: website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+.header {
+  display: flex;
+  align-items: center;
+
+  &::before {
+    content: '';
+    width: 16px;
+    height: 16px;
+    margin-right: 10px;
+    background-size: contain;
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss:36]:
```suggestion
    @include square(16px);
    margin-right: 10px;
    background-size: cover;
```

File: website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+   &::before {
+      background-image: url('/assets/images/alert/error.svg');
+   }
+  }
+}
+
+.header,
+.body {
+   color: $textColor; 
+   font-size: $fontSizeBase;
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.module.scss:68]: `font-size` can be in `.alert` and color is only applies to `.body`

File: website/frontend/src/components/AlertTemplate/AlertTemplate.tsx [hunk 6]
```diff
@@ -1,47 +1,24 @@
...
-    <span style={textStyle}>{message}</span>
-    <button onClick={close} style={buttonStyle} type="button">
-      <X size="18" />
-    </button>
-  </div>
-);
+  return (
+    <div className={styles.alert} style={style}>
+      <Button icon={closeSVG} color="transparent" className={styles.closeBtn} onClick={close} />
+      <div className={clsx(styles.header, styles[options.type || ''])}>{options.type}</div>
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.tsx:15]:
```suggestion
       const { type } = options;
       const headerClassName = clsx(styles.header, styles[type  || '']);
       
      <header className={headerClassName}>{type}</header>
```

File: website/frontend/src/context/alert/Provider.tsx [hunk 7]
```diff
@@ -1,23 +1,24 @@
...
-    minWidth: '300px',
-    margin: 'auto',
-    left: 0,
-    right: 0,
+    width: '390px',
+    left: 'unset',
+    right: '22px',
+    bottom: '5px',
+    alignItems: 'stretch',
+    pointerEvents: 'auto',
```
Comment (nikitayutanov) [inline website/frontend/src/context/alert/Provider.tsx:21]:
```suggestion
    width: '390px',
    right: '22px',
    bottom: '5px',
```

File: website/frontend/src/components/AlertTemplate/AlertTemplate.tsx [hunk 8]
```diff
@@ -1,47 +1,24 @@
...
+      <div className={styles.body}>{message}</div>
+      <div className={styles.footer}>
+        {currentDate.toLocaleDateString()} / {currentDate.toLocaleTimeString()}
+      </div>
```
Comment (nikitayutanov) [inline website/frontend/src/components/AlertTemplate/AlertTemplate.tsx:19]:
```suggestion
      <footer className={styles.footer}>
        {currentDate.toLocaleDateString()} / {currentDate.toLocaleTimeString()}
      </footer>
```

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**